### PR TITLE
Info bug fix and various tweaks

### PR DESCRIPTION
**Pull request by herbig**
## Description

- fix BoxInfo width issue on DAO homepage from adding links
- circle crop ENS avatar image
- remove Favorites menu title (redundant)
- expand favorites menu max height to avoid scrollbars
- make favorites scrollbars on the outside edge of padding rather than inside it
- align top of favorites / account menu dropdowns with each other
- update address helper text to match design
- link ENS/wallet avatar to account on Etherscan
- link name text in DAO Hierarchy to the DAO
- top align transaction input when collapsed
- toggle Show / Hide Executable Code

## Screenshots (if applicable)

Screenshot 2022-12-23 at 5 25 56 PM
Screenshot 2022-12-23 at 5 25 13 PM

### CODE REVIEW

File: src/constants/common.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export const HEADER_HEIGHT = '4rem';
 export const CONTENT_HEIGHT = `calc(100vh - ${HEADER_HEIGHT})`;
 export const BACKGROUND_SEMI_TRANSPARENT = 'black.900-semi-transparent';
+export const ETH_ADDRESS_PLACEHOLDER = '0x1234';
```
Comment (Da-Colon) [inline src/constants/common.ts:4]: versus zeroAddress? just curious why this change?

File: src/components/ui/Header/Avatar.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ function URLAvatar({ url, size }: { size: AvatarSize; url: string }) {
   return (
     <Box w={avatarSizes[size]}>
       <Image
-        className="rounded-full"
+        borderRadius="full"
```
Comment (Da-Colon) [inline src/components/ui/Header/Avatar.tsx:31]: good catch. probably should do a codebase search for `className`. I don't believe there should be any left in the codebase

File: src/components/ui/menus/AccountDisplay/MenuButtonDisplay.tsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import { Box, Flex, Text } from '@chakra-ui/react';
-import { ArrowDown } from '@decent-org/fractal-ui';
```
Comment (Da-Colon) [inline src/components/ui/menus/AccountDisplay/MenuButtonDisplay.tsx:2]: Why do we need to switch to a SVG here versus the Design system icon?

**Review by Da-Colon — COMMENTED** [review]
looks good, got a few questions

File: src/constants/common.ts (hunk 1 above)
Comment (herbig) [reply src/constants/common.ts:4]: @Da-Colon the more I saw the zero address the more it felt pretty weird to me and I started to like what the design had.  I dunno, at the very least consolidating them into this constant will allow us to easily change it back or to whatever we want later on.

File: src/components/ui/Header/Avatar.tsx (hunk 2 above)
Comment (herbig) [reply src/components/ui/Header/Avatar.tsx:31]: yeah I searched for the term and those were the only instances

File: src/components/ui/menus/AccountDisplay/MenuButtonDisplay.tsx (hunk 3 above)
Comment (herbig) [reply src/components/ui/menus/AccountDisplay/MenuButtonDisplay.tsx:2]: ah, yeah we have two different arrows in the menu, the Favorites uses the SVG one, so I used the same here so they'd be the same size.

Though yeah I should use the fractal-UI one for both instead.  I'll update that.

Comment (herbig) [reply src/components/ui/menus/AccountDisplay/MenuButtonDisplay.tsx]: @Da-Colon updated!

File: src/constants/common.ts (hunk 1 above)
Comment (Da-Colon) [reply src/constants/common.ts]: heard. that makes sense to me

**Review by sethhrbek — APPROVED** [review]

**Comment by herbig** [comment]
@Da-Colon I think this PR is good to go now, if I could get an approval!
